docs/app/pages/components/components-sections/tabs/tabs/snippets/app.html

```diff
@@ -13,5 +13,5 @@ <h4>{{ tab.title }}</h4>
             </div>
         </ux-tab>
     </ux-tabset>
-
+    <button class="btn button-primary" (click)="tabset.selectTab(1)">Select Second Tab</button>
```

Comment:
I was just thinking about the example and possibly once in the second tab the button should not be shown, as it has no use there. Same will be for the plunker example.